bundles/org.eclipse.cdt.lsp.clangd/src/org/eclipse/cdt/lsp/clangd/ClangdConfigurationFileManager.java

```diff
@@ -93,7 +106,11 @@ public void setCompilationDatabasePath(IProject project, ICProjectDescription ne
 		}
 	}
 
-	@Override
+	/**
+	 * Enabler for {@link setCompilationDatabasePath}. Can be overriden for customization.
+	 * @param project
+	 * @return true if the database path should be written to .clangd file in the project root.
+	 */
 	public boolean enableSetCompilationDatabasePath(IProject project) {
```

Comment:
I think, since this is no longer part of `ClangdCProjectDescriptionListener` interface, it's enough to make this method protected.